content/_help/verify-your-identity/how-to-take-photos-to-verify-your-identity._en.md

```diff
@@ -0,0 +1,196 @@
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='A person wearing sunglasses. Any face covering results in a failed photo.'
+      image='/assets/img/help/preparing-to-take-your-photo/selfie-accessories-remove.png'
+      text='Only wear accessories, like glasses or hats, that you also wear in your ID photo. Remove other accessories that might block your face.'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='A person with extra light or glare on their face, which results in a failed photo.'
+      image='/assets/img/help/preparing-to-take-your-photo/selfie-avoid-glare.png'
+      text='Take your photo in a well-lit place while avoiding glare.'
+    %}
+  </div>
+</div>
+<div class="grid-row grid-gap">
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='A person with a blurry face, which results in a failed photo.'
+      image='/assets/img/help/preparing-to-take-your-photo/selfie-avoid-blurry.png'
+      text='Remain still to avoid a blurry image.'
+    %}
+  </div>
+  <div class="tablet:grid-col">
+    {%
+      include components/image-example-incorrect.html
+      image-alt-text='A person with their face outside of the central green circle, which results in a failed photo.'
+      image='/assets/img/help/preparing-to-take-your-photo/selfie-fill-green-oval.png'
+      text='Make sure your face fills the entire green oval.'
+    %}
+  </div>
+</div>
+
+### Taking your photo
+<div class="grid-row grid-gap margin-bottom-2">
+  <div class="tablet:grid-col">
+    <img alt="A person with their face in a green oval. A finger taps a checkmark under the face to confirm the photo." src="{{ site.baseurl }}/assets/img/help/preparing-to-take-your-photo/selfie-do-checkmark.png" />
+  </div>
+</div>
+
+The camera will automatically take your photo. Tap the green checkmark to confirm it.
+
+1. Line up your face with the green circle. Make sure your entire face is visible, from your chin to the top of your head.
+2. Tap the check mark to accept the photo.
+3. If the camera tool does not automatically take your photo, position your face so that it fills the green circle and tap the red circle to take a photo.
+
+---
+
+## Troubleshooting
+
+### Ensure the information in your photo is clear
+* Make sure your name, address, and other information on your ID can be easily read.
+* Avoid bright light or glare that may impede the ID scan.
+
+### Consider verifying your identity in person
+* If the information on your ID is not up to date, or if you look very different from your ID photo, verifying your identity online may not work.
+* Select the option to [verify your identity in person](/help/verify-your-identity/verify-your-identity-in-person/) and visit a local United States Postal Service location to verify your identity.
+
```

Comment:
The figma shows a missing section: **Check your device and browser combination**